assemble/bin/accumulo-service

```diff
@@ -69,7 +69,7 @@ function start_service() {
   echo "Starting $service on $host"
 
   if [[ $service == "manager" ]]; then
-    "${bin}/accumulo" org.apache.accumulo.master.state.SetGoalState NORMAL
```

Comment:
That makes sense. I'll add the compatibility classes. There are only a couple main methods in classes that got moved to manager packages.

Reply:
The `KeywordExecutableIT` has a list of classes that were entry points for users that were expected to have a `main` method. `Master` was one of them. However, because we use `KeywordExecutable` to execute these classes, the `main` methods are really optional... we don't need them. We only checked for them for backwards compatibility. That check will fail to catch the backwards compatibility if the IDE automatically renamed the imports for the classes it was checking. The new classes do not need to keep the `main` methods, since they were only there for backwards compatibility (in case somebody wasn't using Accumulo's `Main` class with a keyword.

Reply:
> I'm okay with this change, but there are two considerations that should be made:
> 
> 1. Any renamed file with a `main` method needs to be checked to see if we need to create a compatibility class, in case it was called directly, and

@ctubbsii I pushed changes for this, and a missed default property value update in Property

> 2. Since all the FaTE RepOs were renamed, we need to be sure that on upgrade, we only proceed if there were no outstanding FaTE operations in ZK whatsoever. Otherwise, we won't be able to deserialize and replay them.

I believe this is already happening in [UgradeCoordinator](https://github.com/apache/accumulo/blob/main/server/manager/src/main/java/org/apache/accumulo/master/upgrade/UpgradeCoordinator.java#L144).

Reply:
> The `KeywordExecutableIT` has a list of classes that were entry points for users that were expected to have a `main` method. `Master` was one of them. However, because we use `KeywordExecutable` to execute these classes, the `main` methods are really optional... we don't need them. We only checked for them for backwards compatibility. That check will fail to catch the break in backwards compatibility if the IDE automatically renamed the imports for the classes it was checking. The new classes do not need to keep the `main` methods, since they were only there for backwards compatibility (in case somebody wasn't using Accumulo's `Main` class with a keyword), so they don't need to be kept in the new names.

I'll have a separate PR for renaming classes with Master in the name. If it's ok, I'll take care of removing the main method from the Manager class at that time.